R/logging.R

```diff
@@ -155,18 +155,20 @@ log_problems <- function(notes, control, split, loc, res, bad_only = FALSE) {
   notes
 }
 
-catch_and_log <- function(.expr, ..., bad_only = FALSE, notes) {
+#' @export
+.catch_and_log <- function(.expr, ..., bad_only = FALSE, notes) {
   tune_log(..., type = "info")
   tmp <- catcher(.expr)
   new_notes <- log_problems(notes, ..., tmp, bad_only = bad_only)
   assign("out_notes", new_notes, envir = parent.frame())
   tmp$res
 }
 
-catch_and_log_fit <- function(expr, ..., notes) {
+#' @export
```

Comment:
Should we `@keywords internal` or `@rdname` the rest of these newly exported functions so that they share the `tune_internal_functions` help-file?